src/test/java/com/thealgorithms/sorts/ExchangeSortTest.java

```diff
@@ -0,0 +1,8 @@
+package com.thealgorithms.sorts;
+
+public class ExchangeSortTest extends SortingAlgorithmTest {
```

Comment:
You have to add multiple test cases here like this [https://github.com/TheAlgorithms/Java/blob/master/src/test/java/com/thealgorithms/sorts/BubbleSortTest.java](url)

Reply:
i will work for everycase just like MergeSortTest..!

Reply:
@BamaCharanChhandogi  sir it will work for everycase just like MergeSortTest , @vil02 sir gave suggested me for this ?

Reply:
@BamaCharanChhandogi  i have fixed it as you said so now please merge my PR !